src/panels/config/integrations/integration-panels/zwave_js/zwave_js-custom-param.ts

```diff
@@ -0,0 +1,182 @@
+import { LitElement, html, css, CSSResultGroup, nothing } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import "@material/mwc-button/mwc-button";
+import { mdiCloseCircle } from "@mdi/js";
+import "../../../../../components/ha-textfield";
+import "../../../../../components/ha-select";
+import { HomeAssistant } from "../../../../../types";
+
+@customElement("zwave_js-custom-param")
+class ZWaveJSCustomParam extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property() public deviceId!: string;
+
+  @state() private _customParamNumber = "";
+
+  @state() private _valueSize = "0";
+
+  @state() private _value = "";
+
+  @state() private _valueFormat = "";
+
+  @state() private _error = "";
+
+  protected render() {
+    return html`
+      <div class="custom-config-form">
+        <ha-textfield
+          .label=${this.hass.localize(
+            "ui.panel.config.zwave_js.node_config.parameter"
+          )}
+          .value=${this._customParamNumber}
+          @input=${this._customParamNumberChanged}
+        ></ha-textfield>
+        <ha-select
+          .label=${this.hass.localize(
+            "ui.panel.config.zwave_js.node_config.size"
+          )}
+          .value=${this._valueSize}
+          @selected=${this._customValueSizeChanged}
+        >
+          <ha-list-item value="0"></ha-list-item>
```

Comment:
Is here the label missing or should it be empty when 0?

Reply:
it should be empty